commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -123,17 +123,28 @@ class ContentApiOffersController(contentApiClient: ContentApiClient, capiAgent:
 
   private def renderNative(format: Format, isMulti: Boolean) = Action.async { implicit request =>
```

Comment:
What's this `format` used for? I know it's not apart of the PR, but maybe it's time to remove it.